oeps/oep-0014.rst

```diff
@@ -0,0 +1,188 @@
++---------------+-------------------------------------------+
+| Created       | 2017-01-09                                |
++---------------+-------------------------------------------+
+| `References`  | `ORA PR Discussion`_,                     |
+|               | `Initial Archiving Discussions`_          |
++---------------+-------------------------------------------+
+
+.. _ORA PR Discussion: https://github.com/edx/edx-ora/pull/187
+.. _Initial Archiving Discussions: https://openedx.atlassian.net/wiki/display/IT/Proposed+Github+Deprecation+Process
+
+Abstract
+========
+
+The `edx organization`_ contains a large number of repositories, most of
+which are active and maintained, but some of which are now obsolete. To
+clarify the status of repositories, a process for archiving a
+repository is defined below.
+
+.. _edx organization: https://github.com/edx
+
+Motivation
+==========
+
+Recently openedx.yaml files were added to edX repositories per
+`OEP-2 <oep-0002.rst>`_. In the course of deciding owners for those repositories,
+there was an `ORA PR Discussion`_ about how best to handle deprecated or obsolete
+repositories; in particular, do obsolete repositories need owners, and how can
+repositories be clearly marked as present for archive purposes only?
+
+This discussion resurfaced related to edX's usage of Gemnasium to report
+the usage of third-party libraries that have known security issues. All
+repositories under the edX organization were being monitored, but this added
+noise when trying to understand the number of third-party library updates
+required for actively maintained repositories.
+
+Specification
+=============
+
+When a repository under the `edx organization`_ will no longer be maintained
+because it is no longer in use, the following steps should be followed.
+
+Transfer to New Owner if Interest
+---------------------------------
+First, follow these steps to see if anyone would like to take ownership of
+this repository:
+
+1. Send an email to edx-code@googlegroups.com announcing that the
+repository will be archived, and inquiring if anyone would like to take
+ownership of the repo. If there are no responses after 2 work days, skip
+to `Archive Steps`_.
+
+2. If someone does wish to take ownership of the repository, email the internal
+edX developers mailing list to see if there are any objections. If there are no
+objections after 2 work days, post to edx-code@googlegroups.com that the
+transfer will take place.
+
+3. Create an IT help ticket for the repository to be transferred to the new
+organization.
+
+4. Once the transfer has occurred, create a fork of the transferred
+repository into the `edx organization`_ and follow the `Archive Steps`_ below
+for the forked repo.
+
+Archive Steps
+-------------
+1. Update the README.rst file in the repository to state that it is archived,
+using the `README Archive Statement`_ below.
+
+2. Add "archived: True" to openedx.yaml (creating the file if necessary). Note that
```

Comment:
I went ahead and updated this to reflect the changes that will be merged with #40.